06. 자료구조(스택, 큐)/01. 올바른 괄호/이서인/index.js

```diff
@@ -1,8 +1,20 @@
 function solution(s) {
-  let answer = 'YES';
+  let stack = [];
 
-  return answer;
+  for (const el of s) {
+    if (el === "(") {
+      return stack.push(1);
```

Comment:
여기서 `return stack.push(1);`을 하면 바로 `1`이 출력되고 있어요!
`Array.prototype.push`의 반환 값은 새로운 배열의 길이입니다. [MDN Array.prototype.push](https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Global_Objects/Array/push)

아래를 의도하신 거 같은 느낌이네요~
```suggestion
      stack.push(el);
      continue;
```